logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -194,6 +199,33 @@ class MLSConversationDataSource(
         }
     }
 
+    override suspend fun requestToJoinGroupByExternalCommit(groupID: GroupID, conversationId: QualifiedID): Either<CoreFailure, Unit> {
+        kaliumLogger.d("Requesting to re-join MLS group $groupID via external commit")
+        return wrapApiRequest {
+            conversationApi.fetchGroupInfo(idMapper.toApiModel(conversationId))
+        }.flatMap {
+            mlsClientProvider.getMLSClient().flatMap { mlsClient ->
+                wrapMLSRequest {
+                    mlsClient.joinByExternalCommit(it)
+                }.flatMap { commitBundle ->
+                    sendCommitBundle(groupID, commitBundle)
```

Comment:
This will attempt to call `commitAccepted` which will fail since it's expected that we call `mergePendingGroupFromExternalCommit` instead.